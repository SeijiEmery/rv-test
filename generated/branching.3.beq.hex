0000000    63  04  b5  00  13  03  b0  00  63  04  c5  00  93  03  60  01
0000020    63  04  c5  00  13  0e  10  02  63  04  c5  00  93  0e  c0  02
0000040    13  00  00  00  13  00  00  00  13  00  00  00  13  00  00  00
0000060    13  00  00  00  73  00  10  00                                
0000070
