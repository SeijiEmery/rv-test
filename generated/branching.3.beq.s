beq a0, a1, l1
    addi t1, zero, 11
l1:
    beq a0, a2, l2
    addi t2, zero, 22
l2:
    beq a0, a2, l3
    addi t3, zero, 33
l3:
    beq a0, a2, l4
    addi t4, zero, 44
l4:
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
