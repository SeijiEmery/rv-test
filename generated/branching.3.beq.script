load /x 0 generated/branching.3.beq.hex
writereg 10 10
writereg 11 10
writereg 12 20
writereg 13 -10
writereg 14 -20
writereg 6 101
writereg 7 102
writereg 28 103
writereg 29 104
setpc 0
run 100000
readreg 10
readreg 11
readreg 12
readreg 6
readreg 7
readreg 28
readreg 29