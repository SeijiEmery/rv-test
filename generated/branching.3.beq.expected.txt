tests/branching.test.s:34
R10 = 10
R11 = 10
R12 = 20
R6 = 101
R7 = 22
R28 = 33
R29 = 44
